addons/website/views/snippets.xml

```diff
@@ -387,7 +387,7 @@
     </section>
 </template>
 
-<template id="s_faq_collapse" name="Accordion">
```

Comment:
@fgi-odoo Fabien changed this from "FAQ Collapse" to "Accordion" last year... so we are back to FAQ? :)